store/driver/tikv_driver.go

```diff
@@ -213,7 +214,7 @@ func (d TiKVDriver) OpenWithOptions(path string, options ...Option) (resStore kv
 		tikv.WithCodec(codec),
 	)
 
-	s, err = tikv.NewKVStore(uuid, pdClient, spkv, rpcClient, tikv.WithPDHTTPClient(tlsConfig, etcdAddrs))
+	s, err = tikv.NewKVStore(uuid, pdClient, spkv, &injectTraceClient{Client: rpcClient}, tikv.WithPDHTTPClient(tlsConfig, etcdAddrs))
```

Comment:
IMO we need also wrap [`tikvClient`](https://github.com/pingcap/tidb/blob/9176d206c9b130a5e56f95543740cad66eb469ca/store/copr/store.go#L50-L54), right?

Reply:
No. `tikvClient` is using a internal client returned from `client := s.store.GetTiKVClient()` which has already been wrapped